src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
No that literally means they forced V-Sync in *just* the main menu. Not a global fps lock that also affects the game.